core/src/main/java/com/yahoo/oak/InternalOakMap.java

```diff
@@ -634,6 +634,9 @@ boolean refreshValuePosition(ThreadContext ctx) {
         return ctx.isValueValid();
     }
 
+    public MemoryManager getValuesMemoryManager() {
```

Comment:
Why this change is needed?

Reply:
Yes, since I remove the memory manager reference from the OakMap, for more clarity.

Reply:
OK